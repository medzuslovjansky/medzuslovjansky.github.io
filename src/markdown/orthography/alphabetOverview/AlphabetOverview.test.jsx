import React from "react";
import renderer from "react-test-renderer";
import AlphabetOverview from "./AlphabetOverview";

describe("AlphabetOverview", () => {
  it("matches DOM Snapshot for Latin alphabet", () => {
    const domTree = renderer.create(<AlphabetOverview script="isv-Latn" />).toJSON();
    expect(domTree).toMatchSnapshot();
  });

  it("matches DOM Snapshot for Cyrillic alphabet", () => {
    const domTree = renderer.create(<AlphabetOverview script="isv-Cyrl" />).toJSON();
    expect(domTree).toMatchSnapshot();
  });
});
