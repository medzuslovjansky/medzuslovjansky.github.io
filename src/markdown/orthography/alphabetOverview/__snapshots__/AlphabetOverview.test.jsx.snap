// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlphabetOverview matches DOM Snapshot for Cyrillic alphabet 1`] = `
<table>
  <thead>
    <tr>
      <th>
        Latin
      </th>
      <th>
        Etymological
      </th>
      <th>
        Pronunciation
      </th>
    </tr>
  </thead>
  <tbody
    lang="isv"
  >
    <tr>
      <td
        rowSpan={2}
      >
        А
      </td>
      <td>
        A
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɑ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          a
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        А
      </td>
      <td>
        Å
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɒ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Б
      </td>
      <td>
        B
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          b
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Ц
      </td>
      <td>
        C
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          t͡s
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        Ч
      </td>
      <td>
        Ć
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          t͡ɕ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        Ч
      </td>
      <td>
        Č
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          t͡ʃ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          t͡ʂ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        Д
      </td>
      <td>
        D
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          d
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        Д
      </td>
      <td>
        Ď
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          dʲ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɟ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        ДЖ
      </td>
      <td>
        Đ
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          d͡ʑ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        ДЖ
      </td>
      <td>
        DŽ
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          d͡ʒ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          d͡ʐ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={3}
      >
        Е
      </td>
      <td>
        E
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɛ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          e
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={3}
        style={
          {
            "display": "none",
          }
        }
      >
        Е
      </td>
      <td>
        Ė
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɛ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ǝ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={3}
        style={
          {
            "display": "none",
          }
        }
      >
        Е
      </td>
      <td>
        Ę
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʲæ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Є
      </td>
      <td>
        Ě
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʲɛ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Ф
      </td>
      <td>
        F
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          f
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Г
      </td>
      <td>
        G
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          g
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɦ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Х
      </td>
      <td>
        H
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          x
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        И
      </td>
      <td>
        I
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          i
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɪ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Ј
      </td>
      <td>
        Ј
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          j
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        К
      </td>
      <td>
        K
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          k
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        Л
      </td>
      <td>
        L
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          l
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɫ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        Л
      </td>
      <td>
        Ĺ
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʎ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          l
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Љ
      </td>
      <td>
        LJ
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʎ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          l
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        М
      </td>
      <td>
        M
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          m
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        Н
      </td>
      <td>
        N
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          n
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        Н
      </td>
      <td>
        Ń
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          n
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɲ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Њ
      </td>
      <td>
        NJ
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          nʲ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɲ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        О
      </td>
      <td>
        O
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɔ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          o
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        О
      </td>
      <td>
        Ȯ
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ə
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʌ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        П
      </td>
      <td>
        P
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          p
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        Р
      </td>
      <td>
        R
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          r
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        Р
      </td>
      <td>
        Ŕ
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          rʲ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          r̝
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        С
      </td>
      <td>
        S
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          s
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        С
      </td>
      <td>
        Ś
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɛ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ǝ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Ш
      </td>
      <td>
        Š
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʃ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʂ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        Т
      </td>
      <td>
        T
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          t
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        Т
      </td>
      <td>
        T́
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          tʲ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          c
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        У
      </td>
      <td>
        U
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          u
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        У
      </td>
      <td>
        Ų
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          o
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʊ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        В
      </td>
      <td>
        V
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          v
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʋ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Ы
      </td>
      <td>
        Y
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          i
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɨ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        З
      </td>
      <td>
        Z
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɛ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          z
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        З
      </td>
      <td>
        Ź
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          zʲ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʑ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Ж
      </td>
      <td>
        Ž
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʒ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʐ
        </span>
      </td>
    </tr>
  </tbody>
</table>
`;

exports[`AlphabetOverview matches DOM Snapshot for Latin alphabet 1`] = `
<table>
  <thead>
    <tr>
      <th>
        Latin
      </th>
      <th>
        Etymological
      </th>
      <th>
        Pronunciation
      </th>
    </tr>
  </thead>
  <tbody
    lang="isv"
  >
    <tr>
      <td
        rowSpan={2}
      >
        A
      </td>
      <td>
        A
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɑ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          a
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        A
      </td>
      <td>
        Å
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɒ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        B
      </td>
      <td>
        B
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          b
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        C
      </td>
      <td>
        C
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          t͡s
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        Č
      </td>
      <td>
        Ć
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          t͡ɕ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        Č
      </td>
      <td>
        Č
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          t͡ʃ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          t͡ʂ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        D
      </td>
      <td>
        D
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          d
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        D
      </td>
      <td>
        Ď
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          dʲ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɟ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        DŽ
      </td>
      <td>
        Đ
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          d͡ʑ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        DŽ
      </td>
      <td>
        DŽ
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          d͡ʒ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          d͡ʐ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={3}
      >
        E
      </td>
      <td>
        E
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɛ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          e
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={3}
        style={
          {
            "display": "none",
          }
        }
      >
        E
      </td>
      <td>
        Ė
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɛ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ǝ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={3}
        style={
          {
            "display": "none",
          }
        }
      >
        E
      </td>
      <td>
        Ę
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʲæ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Ě
      </td>
      <td>
        Ě
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʲɛ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        F
      </td>
      <td>
        F
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          f
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        G
      </td>
      <td>
        G
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          g
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɦ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        H
      </td>
      <td>
        H
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          x
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        I
      </td>
      <td>
        I
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          i
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɪ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        J
      </td>
      <td>
        Ј
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          j
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        K
      </td>
      <td>
        K
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          k
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        L
      </td>
      <td>
        L
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          l
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɫ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        L
      </td>
      <td>
        Ĺ
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʎ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          l
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        LJ
      </td>
      <td>
        LJ
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʎ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          l
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        M
      </td>
      <td>
        M
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          m
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        N
      </td>
      <td>
        N
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          n
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        N
      </td>
      <td>
        Ń
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          n
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɲ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        NJ
      </td>
      <td>
        NJ
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          nʲ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɲ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        O
      </td>
      <td>
        O
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɔ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          o
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        O
      </td>
      <td>
        Ȯ
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ə
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʌ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        P
      </td>
      <td>
        P
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          p
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        R
      </td>
      <td>
        R
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          r
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        R
      </td>
      <td>
        Ŕ
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          rʲ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          r̝
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        S
      </td>
      <td>
        S
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          s
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        S
      </td>
      <td>
        Ś
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɛ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ǝ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Š
      </td>
      <td>
        Š
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʃ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʂ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        T
      </td>
      <td>
        T
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          t
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        T
      </td>
      <td>
        T́
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          tʲ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          c
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        U
      </td>
      <td>
        U
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          u
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        U
      </td>
      <td>
        Ų
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          o
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʊ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        V
      </td>
      <td>
        V
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          v
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʋ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Y
      </td>
      <td>
        Y
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          i
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɨ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
      >
        Z
      </td>
      <td>
        Z
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ɛ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          z
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={2}
        style={
          {
            "display": "none",
          }
        }
      >
        Z
      </td>
      <td>
        Ź
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          zʲ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʑ
        </span>
      </td>
    </tr>
    <tr>
      <td
        rowSpan={1}
      >
        Ž
      </td>
      <td>
        Ž
      </td>
      <td>
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʒ
        </span>
         ~
        <span
          className="notranslate"
          lang="art-fonipa"
          translate="no"
        >
          ʐ
        </span>
      </td>
    </tr>
  </tbody>
</table>
`;
